revolv/templates/project/project.html

```diff
@@ -1,240 +1,319 @@
+              <span>OF
+              </span> ${{ project.funding_goal|floatformat:0|intcomma }}
+            </span>
           </span>
           <!-- end .status-text -->
-            <div class="status-indicator desktop-circle">
-              <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="210" data-height="210" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".12">
-            </div>
-            <div class="status-indicator small-circle">
-              <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="175" data-height="175" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".13">
-            </div>
-            <div class="status-indicator tablet-circle">
-              <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="152" data-height="152" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".16">
-            </div>
-            <div class="status-indicator mobile-circle">
-              <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="75" data-height="75" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".12">
-            </div>
+          <div class="status-indicator desktop-circle">
+            <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="210" data-height="210" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".12">
+          </div>
+          <div class="status-indicator small-circle">
+            <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="175" data-height="175" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".13">
+          </div>
+          <div class="status-indicator tablet-circle">
+            <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="152" data-height="152" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".16">
+          </div>
+          <div class="status-indicator mobile-circle">
+            <input type="text" value="{{ project.percent_complete }}" data-min="0" data-width="75" data-height="75" data-bgcolor="#fff" data-fgcolor="#14b1e7" data-max="100" data-readonly="true" data-thickness=".12">
+          </div>
         </div>
       </div>
     </div>
-    <!-- end .banner-section -->
     <div class="info-section">
       <div class="container">
+        <div class="video-main">
+          <div class="row">
+            <div class="video-area col-lg-8 col-md-7">
+              <div class="video">
+                {% if project.video_url %}
+                    <iframe class="project_video youtube-player" allowfullscreen>
```

Comment:
This tag should be on one line, since there's no content within it. For example:

```
<iframe class="project_video youtube-player" allowfullscreen></iframe>
```